keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone.py

```diff
@@ -0,0 +1,518 @@
+
+    return multiply_layer([x, apply_hard_sigmoid(x)])
+
+
+def apply_inverted_res_block(
+    x,
+    expansion,
+    filters,
+    kernel_size,
+    stride,
+    se_ratio,
+    activation,
+    expansion_index,
+    name=None,
+):
+    """An Inverted Residual Block.
+
+    Args:
+        x: input tensor.
+        expansion: integer, the expansion ratio, multiplied with infilters to
+            get the minimum value passed to adjust_channels.
+        filters: integer, number of filters for convolution layer.
+        kernel_size: integer, the kernel size for DepthWise Convolutions.
+        stride: integer, the stride length for DepthWise Convolutions.
+        se_ratio: float, ratio for bottleneck filters. Number of bottleneck
+            filters = filters * se_ratio.
+        activation: the activation layer to use.
+        expansion_index: integer, a unique identification if you want to use
+            expanded convolutions.
+        name: string, layer label.
+
+    Returns:
+        the updated input tensor.
+    """
+    if name is None:
+        name = f"inverted_res_block_{backend.get_uid('inverted_res_block')}"
+
+    shortcut = x
+    prefix = "expanded_conv/"
+    infilters = backend.int_shape(x)[channel_axis]
+
+    if expansion_index:
+        prefix = f"expanded_conv_{expansion_index}"
+
+        x = layers.Conv2D(
+            adjust_channels(infilters * expansion),
+            kernel_size=1,
+            padding="same",
+            use_bias=False,
+            name=prefix + "expand",
+        )(x)
+        x = layers.BatchNormalization(
+            axis=channel_axis,
+            epsilon=1e-3,
+            momentum=0.999,
+            name=prefix + "expand/BatchNorm",
+        )(x)
+        x = activation(x)
+
+    x = layers.DepthwiseConv2D(
+        kernel_size,
+        strides=stride,
+        padding="same" if stride == 1 else "valid",
+        use_bias=False,
+        name=prefix + "depthwise",
+    )(x)
+    x = layers.BatchNormalization(
+        axis=channel_axis,
+        epsilon=1e-3,
+        momentum=0.999,
+        name=prefix + "depthwise/BatchNorm",
+    )(x)
+    x = activation(x)
+
+    if se_ratio:
+        with custom_object_scope({"hard_sigmoid": apply_hard_sigmoid}):
+            x = cv_layers.SqueezeAndExcite2D(
+                filters=adjust_channels(infilters * expansion),
+                ratio=se_ratio,
+                squeeze_activation="relu",
+                excite_activation="hard_sigmoid",
+            )(x)
+
+    x = layers.Conv2D(
+        filters,
+        kernel_size=1,
+        padding="same",
+        use_bias=False,
+        name=prefix + "project",
+    )(x)
+    x = layers.BatchNormalization(
+        axis=channel_axis,
+        epsilon=1e-3,
+        momentum=0.999,
+        name=prefix + "project/BatchNorm",
+    )(x)
+
+    if stride == 1 and infilters == filters:
+        x = layers.Add(name=prefix + "Add")([shortcut, x])
+
+    return x
+
+
+v3Small = {
```

Comment:
These should just be part of the config as in ResNet ([link](https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/backbones/resnet_v2/resnet_v2_backbone.py#L314)).

Reply:
Addressed